portal-web/docroot/html/taglib/ui/input_permissions/page.jsp

```diff
@@ -137,26 +141,29 @@ String modelName = (String)request.getAttribute("liferay-ui:input-permissions:mo
 		</c:choose>
 
 		<script>
-			function <%= uniqueNamespace %>inputPermissionsHideOptions() {
+
+			function <%= uniqueNamespace %>inputPermissionsToggle() {
+				var inputPermissionsOptionsButton = document.getElementById('<%= uniqueNamespace %>inputPermissionsOptionsButton');
+
+				if (inputPermissionsOptionsButton.innerText == '<%= LanguageUtil.get(request, "more-options") %>') {
```

Comment:
Instead of checking the innerText do to one thing or another, better it we check the input value: 
```
var inputPermissionsShowOptionsValue = document.getElementById('<%= uniqueNamespace %>inputPermissionsShowOptions').value;
```

Then, we can simplify a little bit: 
```
inputPermissionsOptionsButton.innerText = inputPermissionsShowOptionsValue ? '<%= LanguageUtil.get(request, "hide-options") %>' : '<%= LanguageUtil.get(request, "more-options") %>';

<%= uniqueNamespace %>togglePermissionsOptions(!inputPermissionsShowOptionsValue);
```
And avoid defining two new functions `hideOptionsBtnClick` and `showOptionsBtnClick`

Reply:
We can do this adding some small modifications, as `inputPermissionsShowOptionsValue` is an String that contains true or false